src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/ProjectLaunchSettingsUIProvider.cs

```diff
@@ -11,45 +11,29 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages
     /// </summary>
     [Export(typeof(ILaunchSettingsUIProvider))]
     [AppliesTo(ProjectCapability.LaunchProfiles)]
-    [Order(0)]              // Lowest priority to allow this to be overridden
+    [Order(Order.Lowest)]              // Lowest priority to allow this to be overridden
```

Comment:
Nit: should we remove the white space here too?

Reply:
Yes, I'll fix that up too, thanks!